config/test/custom-matchers.ts

```diff
@@ -6,7 +6,7 @@ declare global {
     // eslint-disable-next-line @typescript-eslint/no-namespace
     namespace jest {
         interface Matchers<R> {
-            toHaveBeenAnsweredCorrectly(): R;
+            toHaveBeenAnsweredCorrectly(shouldHavePoints?: boolean): R;
```

Comment:
**nit**: This boolean is somewhat non-obvious when looking at the call sites:

```ts 
expect(renderer)toHaveBeenAnsweredCorrectly(false);
```

It makes for more verbose code, but what are your thoughts on:

```ts 
expect(renderer).toHaveBeenAnsweredCorrectly({shouldHavePoints: false});`
```